src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import ball.characteristics.Ball;
+import java.util.Random;
+
+public class Lottery {
+    private final int randomNum = new Random().nextInt(100);
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Ball ball = new Ball(randomNum, colorSupplier.getRandomColor());
+
+    public Ball getRandomBall() {
+        return ball;
+    }
+}
```

Comment:
100 should be a constant
Random should be a private field
random num should a local variable be generated in getRandomBall method. Same for Ball. Right now you only create one unique ball per each lottery instance - it doesn't fulfil our requirements, we want lottery to generate a new Ball every time the method is called